### PR TITLE
[DC-1210] Integration Test for CPT Modifiers CR

### PR DESCRIPTION
**Pull request by lledezma**

### CODE REVIEW

**Comment by hiro-mishima** [comment]
I see `remove_invalid_procedure_source_records.py` being updated here and [this PR for 845](https://github.com/all-of-us/curation/pull/1423/files). Which PR has the right update?

**Comment by lledezma** [comment]
`remove_invalid_procedure_source_records.py` includes the same updates that were implemented in DC-845. I decided to include them here to be able to write the test. Once DC-845 is merged, I will rebase so that this PR only includes the integration test.

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/remove_invalid_procedure_source_records_test.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    (76392641, 5, 5000000, '2011-05-19', TIMESTAMP('2011-05-19'), 38000270, 0, 1, 0, NULL,NULL,'99243', 5000000, NULL),
+    (22888767, 6, 5000000, '2007-12-23', TIMESTAMP('2007-12-28'), 38000270, 0, 1, 0, NULL,NULL,'99243', 5000000, NULL),
+
+    (20074825, 1, 1000000, '2012-09-23', TIMESTAMP('2012-09-23'), 44786631, 0, 1, 0, NULL,NULL,'99458', 2514636, NULL),
+    (12557074, 2, 2000000, '2010-06-21', TIMESTAMP('2010-06-21'), 47876139, 0, 1, 0, NULL,NULL,'45734', 4578984, NULL),
+    (56145455, 3, 3000000, '2011-02-14', TIMESTAMP('2011-02-14'), 45454515, 0, 1, 0, NULL,NULL,'48445', 8545787, NULL)
+
+""")
+
+CONCEPT_IDS_TEMPLATE = JINJA_ENV.from_string("""
```
Comment (hiro-mishima) [inline tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/remove_invalid_procedure_source_records_test.py:38]: There is this function `copy_vocab_tables` available for tests. It loads the vocab tables for you, so you do not have to create vocab tables with dummy values each time.

https://github.com/all-of-us/curation/blob/e9105b626f140d5bfab21130716b0ca1b729c097/tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py#L337

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/remove_invalid_procedure_source_records_test.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    `{{project_id}}.{{dataset_id}}.concept`
+(concept_id, concept_name, domain_id, vocabulary_id, concept_class_id, standard_concept, concept_code, valid_start_date, valid_end_date, invalid_reason)
+
+VALUES
+    (1000000, 'Respiratory-1', 'Procedure', 'ABC',  'CPT4',           'S', '1010882', '2011-02-14', '2011-02-15', NULL ),
+    (2000000, 'Respiratory-2', 'Procedure', 'ABC',  'CPT4 Hierarchy', 'S', '1010882', '2011-02-14', '2011-02-15', NULL),
+    (3000000, 'Respiratory-3', 'Procedure', 'ABC',  'Procedure',      'S', '1010882', '2011-02-14', '2011-02-15', NULL),
+    
+    (4000000, 'Respiratory-4', 'Drug',        'ABC', 'CPT4 Modifier', NULL, '1010882', '2011-02-14', '2011-02-15', NULL),
+    (5000000, 'Respiratory-5', 'Observation', 'ABC', 'CPT4 Modifier', NULL, '1010882', '2011-02-14', '2011-02-15', NULL),
```
Comment (hiro-mishima) [inline tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/remove_invalid_procedure_source_records_test.py:49]: `concept_id` is unique so there should not be two records with `5000000` in real life. However, having two records `5000000` does not break the purpose of this test. So, you can either leave it as it is or fix it.

**Review by hiro-mishima — APPROVED** [review]
Thanks, @lledezma! It looks great overall.
I left two comments, but neither is critical. Either update the test or leave it; either way is fine.
After we finish DC-845, let's merge this one.

**Pull request merged**